docs/pages/tutorial/eas/manage-app-versions.mdx

```diff
@@ -89,6 +68,57 @@ After these steps, the app versions will be synced to EAS Build remotely. You ca
 
 </Collapsible>
 
+## Manually manage app version
```

Comment:
This page is structured in a way where we walk the user through setting up the developer-facing build version values manually and then compare that to how to manage them with EAS automatically. It follows a linear flow. Prior to the updates to the App version management reference page in EAS Build docs, it wasn't clear how to set these values manually or to explain what advantage remote version management provides. That's the context behind why we decided to add this section to the EAS tutorial.

Moving this section below the app version remote causes confusion from a user's perspective about when I've enabled the remote version, why do I need to set these values manually. I think we should avoid doing this right now in this PR since it changes the content and introduces enabling local version support. A proposal I've for upcoming changes (which are out of the scope of this PR):
- Update this section to only mention that manually managing the app version is possible but can be cumbersome.
- Link to the App version management reference page for explanation instead of explaining these concepts here.

Also, the tutorial assumes that the new user persona is installing the EAS CLI and using the latest CLI version. I've created a linear task to follow up on this: https://linear.app/expo/issue/ENG-13013/update-app-version-management-eas-tutorial-chapter-to-account-for after the EAS CLI v11 is released. But for now I don't think we should introduce local version source and the changes below in the tutorial chapter.

Reply:
Sounds reasonable, I'll revert changes to this page